service/collector/chart_config_resource_test.go

```diff
@@ -0,0 +1,49 @@
+package collector
+
+import (
+	"reflect"
+	"testing"
+	"time"
+)
+
+func Test_convertToTime(t *testing.T) {
+	expectedTime, err := time.Parse(time.RFC3339, "2019-12-31T23:59:59Z")
+	if err != nil {
+		t.Errorf("time.Parse err = %v", err)
+	}
+
+	tests := []struct {
+		name         string
+		datetime     string
+		expected     time.Time
+		errorMatcher func(error) bool
+	}{
+		{
+			name:     "case 1: normal timestamp parsing",
+			datetime: "2019-12-31T23:59:59.000",
+			expected: expectedTime,
+		},
+		{
+			name:         "case 2: parsing error since unknown ",
+			datetime:     "2019-12-31T23:59:59Z",
+			errorMatcher: IsInvalidExecution,
+		},
+	}
+	for _, tc := range tests {
+		t.Run(tc.name, func(t *testing.T) {
```

Comment:
```suggestion
		t.Run(strconv.Itoa(i), func(t *testing.T) {
```

Reply:
Before:
```
=== RUN   Test_convertToTime/case_1:_normal_timestamp_parsing
=== RUN   Test_convertToTime/case_2:_parsing_error_since_unknown_
--- PASS: Test_convertToTime (0.00s)
    --- PASS: Test_convertToTime/case_1:_normal_timestamp_parsing (0.00s)
    --- PASS: Test_convertToTime/case_2:_parsing_error_since_unknown_ (0.00s)
```
After:
```
=== RUN   Test_convertToTime
=== RUN   Test_convertToTime/0
=== RUN   Test_convertToTime/1
--- PASS: Test_convertToTime (0.00s)
    --- PASS: Test_convertToTime/0 (0.00s)
    --- PASS: Test_convertToTime/1 (0.00s)
PASS
```

I prefer the former case.